spec/system/settings/image_uploads_spec.rb

```diff
@@ -2,22 +2,24 @@
 
 require 'rails_helper'
 
-RSpec.feature 'Image uploads', type: :feature do
+RSpec.describe 'Image uploads' do
   let(:user) { create(:user) }
   before(:each) { allow(Setting).to receive(:project_name).and_return('Die Lokal-Community!') }
   let(:jwt) { JsonWebToken.encode({ invite_code: 'ONBOARDING_TOKEN', action: 'onboarding' }) }
 
-  scenario 'Upload new onboarding logo' do
+  it 'Upload new onboarding logo' do
     visit onboarding_path(jwt: jwt)
-    expect(page).to have_css('header', text: 'Die Lokal-Community!')
+
+    expect(page).to have_css('header', text: 'DIE LOKAL-COMMUNITY!')
```

Comment:
Change was made because we transform the header to uppercase it and in a real browser the test fails